tools/topology/topology2/cavs-mixin-mixout-efx-hda.conf

```diff
@@ -55,7 +55,12 @@ Object.Pipeline {
 			}
 			Object.Widget.drc.1 {
 				Object.Control.bytes."1" {
-					name '2 Main playback DRC'
+					name 'Post Mixer $ANALOG_PLAYBACK_PCM DRC'
```

Comment:
This small fix could be separate commit.

Reply:
yep.